package.json

```diff
@@ -33,7 +33,6 @@
       "@babel/runtime": "7.26.10",
       "@babel/helpers": "7.26.10",
       "@graphiql/react>@headlessui/react": "2.2.0",
-      "ponder>vite": "5.4.18",
```

Comment:
Addressing a warning emitted during `pnpm install` command execution:
```
 WARN  Issues with peer dependencies found
apps/ensindexer
└─┬ ponder 0.9.27
  └─┬ vite-tsconfig-paths 4.3.1
    └── ✕ unmet peer vite@^6.2.6: found 5.4.18

packages/ponder-schema
└─┬ ponder 0.9.27
  └─┬ vite-tsconfig-paths 4.3.1
    └── ✕ unmet peer vite@^6.2.6: found 5.4.18 in ponder
Done in 5.9s using pnpm v10.8.1
```

Originally, we added this override to fix `pnpm audit` issues, but currently there are none reported even when the override is missing:
```
 pnpm audit --audit-level=low
No known vulnerabilities found
```